main.py

```diff
@@ -14,9 +14,58 @@
 
 """URL routing definitions, and some basic error/warmup handlers."""
 
+#pylint: disable=bad-option-value,wrong-import-position,invalid-name
```

Comment:
No disabling pylint, especially on such a large scale. Figure out how to not do it.

Also please note that we will be soon forcing single-line pylint-disables, so please don't use it on a block of stuff (and, as mentioned, better still not to use it at all). This applies for future PRs too.

Reply:
Hi So I tried adding the sys.path changes below the imports but that definitely doesn't work since the imports look into the sys.path. If we want to do it this way, the sys.path alterations have to occur before the imports and I'm not sure how to do this without disabling the import checks. 
Should I create a new file with this script just to run it before the main.py run?

Reply:
Did you look at other places in the codebase which use sys.path.insert()?

Also, I can understand the use of wrong-import-position on a more specific section of the code, but not the others. See if you can find a way to get rid of those.

Reply:
Just an FYI that this was the PR I was thinking of (in my first comment in this thread): https://github.com/oppia/oppia/pull/9922

Reply:
Fixed!